xml/System.Reflection.Metadata/TypeName.xml

```diff
@@ -135,9 +135,7 @@
           <para>names. For example, given "List&lt;int&gt;", the <see cref="P:System.Reflection.Metadata.TypeName.FullName" /> property will return</para>
           <para>"System.Collections.Generic.List`1[[System.Int32, mscorlib, ...]]".</para>
           <para>For open generic types, the convention is to use a backtick ("`") followed by</para>
-          <para>the arity of the generic type. For example, given "Dictionary&lt;,&gt;", the <see cref="P:System.Reflection.Metadata.TypeName.FullName" /></para>
-          <para>property will return "System.Collections.Generic.Dictionary`2". Given "Dictionary&lt;,&gt;.Enumerator",</para>
-          <para>the <see cref="P:System.Reflection.Metadata.TypeName.FullName" /> property will return "System.Collections.Generic.Dictionary`2+Enumerator".</para>
+          <para>the arity of the generic type. For example, given "Dictionary&lt;,&gt;", the <see cref="P:System.Reflection.Metadata.TypeName.FullName" /> property will return "System.Collections.Generic.Dictionary`2". Given "Dictionary&lt;,&gt;.Enumerator", the <see cref="P:System.Reflection.Metadata.TypeName.FullName" /> property will return "System.Collections.Generic.Dictionary`2+Enumerator".
```

Comment:
This is still an unnecessary `<para>` break, and there are lots of others in this file.

It seems these `<para>` tags might have come from some sort of data import where the source content had hard line wrapping in place at specific line lengths or something, and every line break was turned into a `<para>` unnecessarily. It's reasonable to be more aggressive in collapsing `<para>` tags together in this PR, only leaving them in place where it's very clear that a new paragraph was intended.